crowbar_framework/lib/crowbar/upgrade_timeouts.rb

```diff
@@ -0,0 +1,37 @@
+require "yaml"
+
+module Crowbar
+  class UpgradeTimeouts
+    def values
+      @timeouts_config = begin
+        YAML.load_file("/etc/crowbar/upgrade_timeouts.yml")
```

Comment:
> AFAIU these get recreated with every API request, right?

Hm, actually this seems to be wrong for the Api::Upgrade case. It's all class methods ...